src/hooks/useIsAmbireWallet.ts

```diff
@@ -0,0 +1,8 @@
+import { useWalletMetaData } from '@cow/modules/wallet'
+import { getIsAmbireWallet } from '@src/custom/connection/utils'
+
+export default function useIsAmbireWallet(): boolean {
+  const { walletName } = useWalletMetaData()
+
+  return Boolean(getIsAmbireWallet(walletName))
```

Comment:
Redundant cast to boolean. `getIsAmbireWallet` already has `boolean` return type